test/system_test.py

```diff
@@ -105,23 +88,11 @@ def spack_devbuild_and_test(spec: str,
 
     command = 'dev-build'
 
-    if split_phases:
-        log_with_spack(f'spack {command} --until build --test=root -n {spec}',
-                       'system_test',
-                       log_filename,
-                       cwd=cwd,
-                       srun=True)
-        log_with_spack(f'spack {command} --dont-restage --test=root -n {spec}',
-                       'system_test',
-                       log_filename,
-                       cwd=cwd,
-                       srun=False)
-    else:
-        log_with_spack(f'spack {command} --test=root -n {spec}',
-                       'system_test',
-                       log_filename,
-                       cwd=cwd,
-                       srun=True)
+    log_with_spack(f'spack {command} --test=root -n {spec}',
+                   'system_test',
+                   log_filename,
+                   cwd=cwd,
```

Comment:
the command `spack_devbuild_and_test` in  not used anymore.

You could delete it entirely.

Reply:
Out of the scope of this PR. But it's a good topic for an other PR. I'll create one.